src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/DeciderConfig.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.RcaConf;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DeciderConfig {
+
+    private static final String CACHE_BOUNDS_CONFIG_NAME = "cache-bounds";
+    private static final String CACHE_CONFIG_NAME = "cache-type";
+    private static final String WORKLOAD_CONFIG_NAME = "workload-type";
+    private static final String PRIORITY_ORDER_CONFIG_NAME = "priority-order";
+    private static final String FIELD_DATA_CACHE_UPPER_BOUND_NAME = "field-data-cache-upper-bound";
+    private static final String SHARD_REQUEST_CACHE_UPPER_BOUND_NAME = "shard-request-cache-upper-bound";
+    private static final double DEFAULT_FIELD_DATA_CACHE_UPPER_BOUND = 0.4;
+    private static final double DEFAULT_SHARD_REQUEST_CACHE_UPPER_BOUND = 0.05;
```

Comment:
We can split cache bounds and queue bounds into their respective config objects. They will still get read from the decider base class so all deciders have it. But this will help organize the overall structure. (`QueueConfig` will get passed to `ModifyQueueAction` etc. )
We can do this as a separate PR.

Reply:
So as suggested cache bounds / queue bounds etc. should be read as a single object from the config file? This can be done when the queue related configs are added in the config file.